ext/standard/tests/math/pow_variation1.phpt

```diff
@@ -62,18 +62,20 @@ $inputs = array(
 /*20*/ "abcxyz",
        'abcxyz',
        $heredoc,
+       '5.5',
+       '2',
```

Comment:
```suggestion
       '5.5',
       '2',
```

Frankly we don't care about testing "different ways to write strings"

Reply:
As this is the function of power, I think it's relevant that it works fine for numerical strings. I want to propose later setting up parameter types for this function, but wanted to fix the tests first.

Reply:
I was talking about the heredoc and usage of single quote instead of double. The numeric string cases make sense, although you are missing a numeric string in scientific notation.

Reply:
I understand, thank you. Removed the heredoc string and added numeric string in scientific notation 😉